### DISCUSSION TITLE
File picker like menu for global search

### DISCUSSION THREAD

**Discussion by KittyBorgX** [helix-editor/helix#7258] [Q&A] [answered]
Currently, global search in a project (`space + /`) works similar to vanilla's vim's normal search and the menu appears after I pressed `ENTER` after typing the search query in. Is there any way in which there is interactive search, similar to Neovim's Telescope Plugin?

**Comment by the-mikedavis** [comment, marked as answer]
Not currently. This is covered by https://github.com/helix-editor/helix/issues/196

**Reply by KittyBorgX** [reply]
Thanks!
